src/models/index.js

```diff
@@ -1,19 +1,20 @@
 import fs from 'fs';
 import path from 'path';
 import Sequelize from 'sequelize';
-import dbConfig from '../config/config';
+import { development, production, test } from '../config/config';
+
+const environment = {
+  development,
+  production,
+  test
+};
 
 const basename = path.basename(__filename);
 const env = process.env.NODE_ENV || 'development';
-const config = dbConfig[env];
+const config = environment[env];
```

Comment:
why did you change here?
I think to maintain the way the file was we can do`
```
import { development, test, production } from '../config/config';
const dbConfig = { development, test, production };
```
```suggestion
const config = dbConfig[env];
```